snips/config.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Snips.AI",
-  "version": "0.6.5-p1",
+  "version": "0.51.14",
```

Comment:
This is the version number of the add-on. We should just bump it to 0.7

Reply:
0.6.5 was also the version number of snips at one point. But that makes sense to keep our own so we can iterate between snips releases if we wanted.

Reply:
We should not couple the version number. We should be able to update the add-on without updating Snips.